pkg/action/package_test.go

```diff
@@ -23,30 +23,8 @@ import (
 	"testing"
 
 	"helm.sh/helm/v3/internal/test/ensure"
-	"helm.sh/helm/v3/pkg/chart"
 )
 
-func TestSetVersion(t *testing.T) {
```

Comment:
can you please add this test back. test cases should not be removed. If it needs to be modified as part of a bug fix, then let's fix up the test case, but we should not be removing test coverage unless absolutely necessary.

Reply:
In the latest revision, the `setVersion` method has been moved out and a new verification method has been added, so I removed this exported `TestSetVersion`, is it necessary to add a unit test for `validateVersion`

My personal opinion is not necessary

Reply:
Yes, a test case is necessary to ensure regressions are not introduced in the future. Especially something that can be easily tested like validateVersion.

I’d write test cases for the most common use cases to ensure the behaviour works as expected.

Reply:
It makes sense, but I have a strange feeling that the unit test here feels like testing the semver library, like the responsibility of the library itself

Reply:
@wawa0210 In this case we are testing the Helm experience. What if something breaks in the SemVer library? We want to catch that before it bubbles up to Helm users. On a widely used and stable project like this it's important to catch issues early.